src/main/java/mods/railcraft/common/blocks/machine/worldspike/BlockWorldspikePoint.java

```diff
@@ -84,20 +86,29 @@ public float getBlockHardness(IBlockState state, World worldIn, BlockPos pos) {
 
     @Override
     public float getExplosionResistance(World world, BlockPos pos, Entity exploder, Explosion explosion) {
-//        return 60f * 3f / 5f;
         return 90f;
     }
 
     @Override
     public void defineRecipes() {
         if (RailcraftConfig.canCraftStandardWorldspikes()) {
             CraftingPlugin.addRecipe(getStack(),
-                    " p ",
-                    " o ",
-                    "ogo",
-                    'g', "ingotGold",
-                    'p', Items.ENDER_PEARL,
-                    'o', new ItemStack(Blocks.OBSIDIAN));
+                " p ",
+                " o ",
+                "ogo",
+                'g', "ingotGold",
+                'p', Items.ENDER_PEARL,
+                'o', new ItemStack(Blocks.OBSIDIAN));
         }
     }
+
+    @Override
+    public String getUnlocalizedName() {
+        return super.getUnlocalizedName() + ".name";
```

Comment:
This is wrong because it is not from tiles. Will change in localization files then.